libs/payments/stripe/src/lib/accountCustomer/accountCustomer.repository.ts

```diff
@@ -0,0 +1,129 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  // AccountCustomerUpdate,
+  AccountDatabase,
+  NewAccountCustomer,
+} from '@fxa/shared/db/mysql/account';
+
+// import {
+//   AccountCustomerManagerError,
+//   AccountCustomerNotUpdatedError,
+// } from './accountCustomer.error';
+
+// export async function accountExists(db: AccountDatabase, uid: Buffer) {
```

Comment:
Should `accountExists` be moved to `account.repository.ts`?

Reply:
For posterity, this question was answered in Slack - `accountExists` will not be a part of the new structure.